pingpong/ai.py

```
@@ -3721,12 +3751,64 @@ async def _tts_receive_audio(
+                        else:
+                            assert tts_api_key is not None
+                            assert tts_voice_id is not None
+                            _tts_client = ElevenLabsStreamingTTS(
+                                tts_api_key, tts_voice_id
+                            )
+                            await _tts_client.connect()
+                        handler.enqueue_audio_started()
+                        _tts_audio_task = asyncio.create_task(
+                            _tts_receive_audio(_tts_client)
+                        )
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)